cmd/bypass4netns/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+// This code is copied from 'runc(https://github.com/opencontainers/runc/blob/v1.1.0/contrib/cmd/seccompagent/seccompagent.go)'
+// The code is licensed under Apach-2.0 License
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/rootless-containers/bypass4netns/pkg/bypass4netns"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	socketFile string
+	pidFile    string
+)
+
+func main() {
+	xdg_runtime_dir := os.Getenv("XDG_RUNTIME_DIR")
+	flag.StringVar(&socketFile, "socketfile", xdg_runtime_dir+"/bypass4netns.sock", "Socket file")
```

Comment:
```suggestion
	flag.StringVar(&socketFile, "socket", filepath.Join(xdgRuntimeDir, "bypass4netns.sock"), "Socket file")
```

Reply:
And we can `panic("$XDG_RUNTIME_DIR needs to be set")` when `xdgRuntimeDir == ""`